### PR TITLE
[SOFIA-LAB] Vmware 7 support

### PR DESCRIPTION
**Pull request by nvazquez**
- Sofia:
   - Added 70u1 templates on `all` file in the `sofia-lab` branch of the trilliangroup_vars repo
   - vc_nested_service_offering: "Nested VC Server"

This solution for 4.14 uses a nested vCenter solution (template = "vCenter7-Nested-New")

**Upgrade to 4.15:**
Since 4.15 we will be able to use the vCenter template directly with the deploy-as-is feature: `set vc_template: "vCenter7 Appliance"`

### CODE REVIEW

File: Ansible/group_vars/all.sample [hunk 1]
```diff
@@ -748,6 +751,7 @@ def_vmware_templates:
     esxi_template: "na"
     vc_template: "na"
 def_vc_service_offering: "VC server"
+def_vc_nested_service_offering: "Nested VC Server"
```
Comment (rhtyd) [inline Ansible/group_vars/all.sample:754]: Is this service offering for RAM >=14 GB (12GB for vcenter and 1-2 GB for ESXi?)

**Review by rhtyd — APPROVED** [review]
High level lgtm, we need to run and test it - as well as run regression test matrix in case changes here break other env. Andrija/David best to review/test this. (treat my review as moral support than anything else :D)

File: Ansible/group_vars/all.sample (hunk 1 above)
Comment (nvazquez) [reply Ansible/group_vars/all.sample:754]: Yes - this one is 15GB RAM x 4CPUs

**Comment by rhtyd** [comment]
@nvazquez can this be merged ?

**Comment by nvazquez** [comment]
Rebased `sofia-lab` as I was getting SSH issues
````
====Trying SSH Connection: Host:10.0.32.252 User:root                                   Port:22 RetryCnt:60===
SshClient: Exception under createConnection: ['Traceback (most recent call last):\n', '  File "/usr/lib/python2.7/site-packages/marvin/sshClient.py", line 122, in createConnection\n    allow_agent=False)\n', '  File "/usr/lib/python2.7/site-packages/paramiko/client.py", line 338, in connect\n    t.start_client()\n', '  File "/usr/lib/python2.7/site-packages/paramiko/transport.py", line 492, in start_client\n    raise e\n', 'ValueError: CTR mode needs counter parameter, not IV\n']
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/marvin/sshClient.py", line 122, in createConnection
    allow_agent=False)
  File "/usr/lib/python2.7/site-packages/paramiko/client.py", line 338, in connect
    t.start_client()
  File "/usr/lib/python2.7/site-packages/paramiko/transport.py", line 492, in start_client
    raise e
ValueError: CTR mode needs counter parameter, not IV
````

**Comment by nvazquez** [comment]
@rhtyd starting a new round of tests after rebasing - will report if the SSH error is fixed and this can be merged

**Comment by nvazquez** [comment]
Rebased fixed the SSH issue, merging

**Pull request merged**